website/docs/ephemeral-resources/cognito_identity_openid_token_for_developer_identity.markdown

```diff
@@ -0,0 +1,53 @@
+---
+subcategory: "Cognito Identity"
+layout: "aws"
+page_title: "AWS: aws_cognito_identity_openid_token_for_developer_identity"
+description: |-
+  Terraform ephemeral resource for managing an AWS Cognito Identity Open ID Token for Developer Identity.
+---
+
+
+# Ephemeral: aws_cognito_identity_openid_token_for_developer_identity
+
+Terraform ephemeral resource for managing an AWS Cognito Identity Open ID Token for Developer Identity.
+
+## Example Usage
+
+### Basic Usage
+
+```terraform
+data "aws_cognito_identity_pool" "example" {
+  identity_pool_name = "test pool"
+}
+
+ephemeral "aws_cognito_identity_openid_token_for_developer_identity" "example" {
+  identity_pool_id = data.aws_cognito_identity_pool.example.id
+  logins = {
+	  "login.mycompany.myapp": "USER_IDENTIFIER"
+  }
```

Comment:
[AWS Docs](https://aws.amazon.com/blogs/mobile/understanding-amazon-cognito-authentication-part-2-developer-authenticated-identities/) give more examples of login. Does it make sense to call all these out
```
"Logins": {
    "graph.facebook.com": "FB_TOKEN",
    "accounts.google.com": "GOOGLE_TOKEN",
    "www.amazon.com": "AMZN_TOKEN",
    "login.mycompany.myapp": "USER_IDENTIFIER"
}
```